Sources/ServiceKit/Environment/Environment.swift

```diff
@@ -42,6 +43,13 @@ public struct Environment: Equatable {
         return ProcessInfo.processInfo.environment[key]
     }
 
+    /// Gets a key from the proccess environment
+    ///
+    ///     Environment.detect().DATABASE_USER
+    public subscript(dynamicMember member: String) -> String? {
+        return ProcessInfo.processInfo.environment[member]
```

Comment:
What's the overhead of accessing this? Could we try caching?

Reply:
Process environment should only be accessed during boot-time, so I think performance doesn't really matter here. Caching would increase memory usage which would be a downside even after boot.

Reply:
Comment withdrawn!